### ISSUE TITLE
show_weights with OneVsRestClassifier

### ISSUE THREAD

**Issue by s-j** [TeamHG-Memex/eli5#106]
Hi guys, I really like this tool! I have a pipeline say

```
mlb = MultiLabelBinarizer()
y_train = mlb.fit_transform(y_train)
vec = TfidfVectorizer(ngram_range=(1, 2), stop_words='english')
clf = OneVsRestClassifier(LogisticRegressionCV())
pipeline = make_pipeline(vec, clf)
pipeline.fit(X_train, y_train)
```
`show_prediction` works neatly, but I run into 'LogisticRegressionCV' object has no attribute 'classes_' when calling `eli5.show_weights(clf.estimator, vec=vec, target_names=mlb.classes_)` or unsupported class if I use `clf` directly.

Is it possible to work around this problem or do you plan adding support for this soon?

Cheers!
Simon

**Comment by kmike** [comment]
Could you try passing clf directly, not clf.estimator? eli5 should support OneVsRestClassifier directly; it is a bug if it doesn't work. 

By the way, is there a reason you're using OneVsRestClassifier with LogisticRegression? If I'm not mistaken, LR uses OvR internally by default.

**Comment by s-j** [comment]
Wow, thanks for a quick reply!

I am new to scikit-learn, but to my understanding OneVsRestClassifier adds multi-label support in comparison to pure multi-class. Have I misunderstood this?

Passing clf directly returns "Error: estimator OneVsRestClassifier(estimator=LogisticRegressionCV...) is not supported". I see that there is a dispatch trick in show_prediction, by passing clf as clf.estimator.__class__.

This actually did work for me:

```
setattr(clf.estimator, 'classes_', clf.classes_)
setattr(clf.estimator, 'coef_', clf.coef_)
setattr(clf.estimator, 'intercept_', clf.intercept_)
eli5.show_weights(clf.estimator, vec=vec, target_names=mlb.classes_)
```

**Comment by kmike** [comment]
Ah, right, it looks like OvR is indeed only supported for explain_prediction, not for explain_weights. This looks like an oversight, thanks for the heads up! It must be possible to implement a similar fix for explain_weights. Do you want to send a PR? Alternatively, I can fix that tomorrow.

Yeah, a good point about OneVsRestClassifier and multi-label support; I missed that.

**Comment by s-j** [comment]
Great, thanks!

**Comment by s-j** [comment]
(I would appreciate if you could do it. No rush.)

**Comment by kmike** [comment]
Your example should be fixed by https://github.com/TeamHG-Memex/eli5/pull/108. Thanks for the report!
